embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/EmbraceDeliveryRetryManager.kt

```diff
@@ -84,25 +96,25 @@ internal class EmbraceDeliveryRetryManager(
     }
 
     /**
-     * Returns true if there is an active pending retry task
+     * Returns true if the number of retries for the endpoint is below the limit.
      */
+    private fun isBelowRetryLimit(endpoint: Endpoint): Boolean {
```

Comment:
Can this just be returned by `FailedApiCallsPerEndpoint` instead of exposing the count?

Reply:
Yes, I moved this to FailedApiCallsPerEndpoint.